### PR TITLE
[SMALLFIX] Improve the logic to load metadata for root

### PR DESCRIPTION
**Pull request by peisun1115**

### CODE REVIEW

**Comment by peisun1115** [comment]
@gpang 

@apc999

**Comment by peisun1115** [comment]
@apc999   https://github.com/Alluxio/alluxio/pull/3436 hides logs related to this

File: core/server/src/test/java/alluxio/master/file/FileSystemMasterTest.java [hunk 1]
```diff
@@ -142,7 +142,9 @@ public void after() {
    */
   @Before
   public void before() throws Exception {
-    mUnderFS = mTestFolder.newFolder().getAbsolutePath();
+    // This makes sure that the mount point of the UFS corresponding to the Alluxio root ("/")
+    // doesn't exist by default (helps loadRootTest).
+    mUnderFS = mTestFolder.newFolder().getAbsolutePath() + "/underFs";
```
Comment (apc999) [inline core/server/src/test/java/alluxio/master/file/FileSystemMasterTest.java]: NIT: `PathUtils.concatPath`

**Comment by apc999** [comment]
one minor comment left.

File: core/server/src/test/java/alluxio/master/file/FileSystemMasterTest.java [hunk 2]
```diff
@@ -41,6 +41,7 @@
 import alluxio.thrift.FileSystemCommand;
 import alluxio.util.IdUtils;
 import alluxio.util.io.FileUtils;
+import alluxio.util.io.PathUtils;
 import alluxio.wire.FileBlockInfo;
 import alluxio.wire.FileInfo;
 import alluxio.wire.WorkerNetAddress;
```
Comment (peisun1115) [inline core/server/src/test/java/alluxio/master/file/FileSystemMasterTest.java]: Done.

**Comment by peisun1115** [comment]
@apc999 @gpang  more comments?

File: core/server/src/test/java/alluxio/master/file/FileSystemMasterTest.java [hunk 3]
```diff
@@ -142,7 +143,9 @@ public void after() {
    */
   @Before
   public void before() throws Exception {
-    mUnderFS = mTestFolder.newFolder().getAbsolutePath();
+    // This makes sure that the mount point of the UFS corresponding to the Alluxio root ("/")
+    // doesn't exist by default (helps loadRootTest).
+    mUnderFS = PathUtils.concatPath(mTestFolder.newFolder().getAbsolutePath() + "underFs");
```
Comment (apc999) [inline core/server/src/test/java/alluxio/master/file/FileSystemMasterTest.java]: `PathUtils.concatPath(mTestFolder.newFolder().getAbsolutePath(), "underFs"`?

**Comment by apc999** [comment]
one question left

File: core/server/src/test/java/alluxio/master/file/FileSystemMasterTest.java (hunk 3 above)
Comment (peisun1115) [inline core/server/src/test/java/alluxio/master/file/FileSystemMasterTest.java]: sorry, typo. Fixed

**Comment by gpang** [comment]
LGTM

**Comment by apc999** [comment]
LGTM

**Pull request merged**